### ISSUE TITLE
Error while transforming a person

### ISSUE THREAD

**Issue by julienfastre** [secit-pl/json-ld-bundle#2]
I encounter this error : `Type error: Argument 1 passed to SecIT\SchemaOrg::toJsonLd() must be an instance of SecIT\SchemaOrg\Mapping\Type\Thing, instance of SecIT\SchemaOrg\Mapping\Type\Person given, called in /var/www/app/vendor/secit-pl/json-ld-bundle/Service/JsonLd.php on line 70`

I think that this comes from the fact that `SecIT\SchemaOrg\Mapping\Type\Person` (from https://github.com/secit-pl/schema-org) does not extends `Thing`

**Comment by secit-pl** [comment]
For which release is this issue? Could you confirm that the latest release 2.2.0 is also affected to this error?

**Comment by julienfastre** [comment]
I am sorry, I used by error 2.1 version.

Can you confirm that this is fixed in next release ?

I also encounter this bug, which is blocking :
https://github.com/secit-pl/schema-org/issues/2

**Comment by secit-pl** [comment]
The `Type error` should be fixed in the release 3.3.1 of the https://github.com/secit-pl/schema-org. Could you confirm that it's ok now?

https://github.com/secit-pl/schema-org/issues/2 -> I'll try to check this out later

**Comment by secit-pl** [comment]
Tested on https://github.com/secit-pl/schema-org at release 3.3.2:

PHP code:

```php
$schemaOrg = new \SecIT\SchemaOrg();
$person = new Type\Person();
$person->setName(new Property\Name(new DataType\TextType('Jon Smith')))
    ->setEmail(new Property\Email(new DataType\TextType('mail@example.com')));

echo $schemaOrg->toJsonLd($person);
```

The output:

```html
<script type="application/ld+json">{"@context":"http:\/\/schema.org","@type":"Person","email":"mail@example.com","name":"Jon Smith"}</script>
```

Closing the issue.
